systemtest/src/test/java/io/strimzi/systemtest/operators/MultipleClusterOperatorsST.java

```diff
@@ -0,0 +1,247 @@
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+@Tag(REGRESSION)
+public class MultipleClusterOperatorsST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(MultipleClusterOperatorsST.class);
+
+    public static final String DEFAULT_NAMESPACE = "multiple-co-cluster-test";
+    public static final String FIRST_NAMESPACE = "first-co-namespace";
+    public static final String SECOND_NAMESPACE = "second-co-namespace";
+
+    public static final String FIRST_CO_NAME = "first-" + Constants.STRIMZI_DEPLOYMENT_NAME;
+    public static final String SECOND_CO_NAME = "second-" + Constants.STRIMZI_DEPLOYMENT_NAME;
+
+    public static final EnvVar FIRST_CO_SELECTOR_ENV = new EnvVar("STRIMZI_CUSTOM_RESOURCE_SELECTOR", "co-name=" + FIRST_CO_NAME, null);
+    public static final EnvVar SECOND_CO_SELECTOR_ENV = new EnvVar("STRIMZI_CUSTOM_RESOURCE_SELECTOR", "co-name=" + SECOND_CO_NAME, null);
+
+    public static final Map<String, String> FIRST_CO_SELECTOR = Collections.singletonMap("co-name", FIRST_CO_NAME);
+    public static final Map<String, String> SECOND_CO_SELECTOR = Collections.singletonMap("co-name", SECOND_CO_NAME);
+
+    @Test
+    void testMultipleCOsWithDifferentCustomResourceSelector() {
+        deployCOInNamespace(FIRST_CO_NAME, DEFAULT_NAMESPACE, FIRST_CO_SELECTOR_ENV, false);
+        deployCOInNamespace(SECOND_CO_NAME, DEFAULT_NAMESPACE, SECOND_CO_SELECTOR_ENV, false);
+
+        testMultipleCOsWithDifferentCRSelectors();
+    }
+
+    @Test
+    void testMultipleCOsInDifferentNamespaces() {
+        deployCOInNamespace(FIRST_CO_NAME, FIRST_NAMESPACE, FIRST_CO_SELECTOR_ENV, true);
+        deployCOInNamespace(SECOND_CO_NAME, SECOND_NAMESPACE, SECOND_CO_SELECTOR_ENV, true);
+
+        cluster.setNamespace(DEFAULT_NAMESPACE);
+
+        testMultipleCOsWithDifferentCRSelectors();
+    }
+
+    void testMultipleCOsWithDifferentCRSelectors() {
+        String producerName = "hello-world-producer";
+        String consumerName = "hello-world-consumer";
+
+        LOGGER.info("Deploying Kafka with CR selector pointing at first CO");
+        KafkaResource.kafkaWithoutWait(KafkaResource.kafkaEphemeral(clusterName, 3, 3).build());
+
+        PodUtils.waitUntilPodStabilityReplicasCount(clusterName, 0);
+
+        LOGGER.info("Add selector for {} into Kafka CR", FIRST_CO_NAME);
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> kafka.getMetadata().setLabels(FIRST_CO_SELECTOR));
+        KafkaUtils.waitForKafkaReady(clusterName);
+
+        LOGGER.info("Deploying topic with different CR selector than Kafka has");
+        KafkaTopicResource.createAndWaitForReadiness(KafkaTopicResource.topic(clusterName, TOPIC_NAME)
+            .editOrNewMetadata()
+                .addToLabels(SECOND_CO_SELECTOR)
+            .endMetadata()
+            .build());
```

Comment:
What is expected to happen from this? The Topic Operator doe snot follow the selector. It follows only the `strimzi.io/cluster` label.